css/base.css

```diff
@@ -0,0 +1,401 @@
+/* Open Sans Font */
+@font-face {
+  font-family: 'open_sansbold';
```

Comment:
Please review this:
http://www.anotheruiguy.com/ux-design-dev/_book/web-fonts/README.html